diff --git a/node_modules/@bazel/typescript/internal/tsc_wrapped/tsconfig.js b/node_modules/@bazel/typescript/internal/tsc_wrapped/tsconfig.js
index 62f83dc..bfd6362 100755
--- a/node_modules/@bazel/typescript/internal/tsc_wrapped/tsconfig.js
+++ b/node_modules/@bazel/typescript/internal/tsc_wrapped/tsconfig.js
@@ -3,6 +3,7 @@
  * @license
  * Copyright 2017 The Bazel Authors. All rights reserved.
  *
+ *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  *
@@ -91,7 +92,7 @@ function parseTsconfig(tsconfigFile, host = ts.sys) {
         return [null, errors, { target }];
     }
     // Override the devmode target if devmodeTargetOverride is set
-    if (bazelOpts.es5Mode && bazelOpts.devmodeTargetOverride) {
+    if (/*bazelOpts.es5Mode && */bazelOpts.devmodeTargetOverride) {
         switch (bazelOpts.devmodeTargetOverride.toLowerCase()) {
             case 'es3':
                 options.target = ts.ScriptTarget.ES3;
