diff --git a/node_modules/@deepkit/type/dist/cjs/src/model.js b/node_modules/@deepkit/type/dist/cjs/src/model.js
index 5f1fec9..f6cdbce 100644
--- a/node_modules/@deepkit/type/dist/cjs/src/model.js
+++ b/node_modules/@deepkit/type/dist/cjs/src/model.js
@@ -20,11 +20,9 @@ var UnpopulatedCheck;
     UnpopulatedCheck[UnpopulatedCheck["Throw"] = 1] = "Throw";
     UnpopulatedCheck[UnpopulatedCheck["ReturnSymbol"] = 2] = "ReturnSymbol";
 })(UnpopulatedCheck = exports.UnpopulatedCheck || (exports.UnpopulatedCheck = {}));
-const global = {
-    RegisteredEntities: {},
-    unpopulatedCheck: UnpopulatedCheck.Throw,
-    enableForwardRefDetection: true,
-};
+global.RegisteredEntities = {};
+global.unpopulatedCheck = UnpopulatedCheck.Throw;
+global.enableForwardRefDetection = true;
 function getGlobalStore() {
     return global;
 }
